pages.pt-BR/common/axel.md

```diff
@@ -0,0 +1,25 @@
+# axel
+
+> Acelerador de downloads.
+> Suporta HTTP, HTTPS, e FTP.
+> Página Oficial: <https://github.com/axel-download-accelerator/axel>.
+
+- Fazer download de uma URL para um arquivo:
+
+`axel {{url}}`
+
+- Fazer download especificando o nome do arquivo destino:
```

Comment:
Maybe "arquivo de destino"? (Disregard my suggestion if this is usual terminology in pt-BR.)

Reply:
"Arquivo de destino" reads better. 
And a use of that can already be seen [here](pages.pt-BR/common/cat.md)